src/components/Tool/Appeal/List/ContactsList/ContactsList.tsx

```diff
@@ -28,10 +29,13 @@ const useStyles = makeStyles()(() => ({
   contactHeader: {
     padding: theme.spacing(1, 2),
   },
+  excludedHeader: {
+    padding: theme.spacing(1, 2, 1, 0),
+  },
   givingHeader: {
-    padding: '8px 16px 8px 30px',
+    padding: theme.spacing(1, 2, 1, 0),
     [theme.breakpoints.down('md')]: {
-      padding: '8px 16px 8px 8px',
+      padding: theme.spacing(1, 2, 1, 0),
```

Comment:
Can this breakpoint be removed since it's the same as the default styles?

Reply:
yes, I meant to remove it.